src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -264,7 +265,17 @@ private static void processItems(Document document, Node itemsNode, Property pro
             }
 
             if (StringUtils.equalsIgnoreCase("multifield", property.getType())) {
-                optionNode.setAttribute(Constants.PROPERTY_NAME, "./" + item.getField());
+                getPrimaryFieldName(item).ifPresent(name -> {
```

Comment:
The updated code is adding multifield images to the sling model and dialog correctly, but the generated HTML isn't quite right.

```
<p>Path: ${item.path} | Label: ${item.label} | Link Icon: ${item.linkIcon}</p>
```

`item.linkIcon` doesnt render on the page since it's an Image object.  Our standard default rendered HTML for an image looks like this: `<img src="${demoCompModel.imageTest.src}"/>`

Reply:
Aha, didn't even notice that bit of html. Will fix

Reply:
@HitmanInWis I've updated this to at least dump the value of Image.src as a string, since the inline multifield item is rendered as a single, pipe-delimited line.